### PR TITLE
added makefile

### PR DESCRIPTION
**Pull request by rcnijzink**
Addition of makefile as first trial

### CODE REVIEW

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 1]
```diff
@@ -149,13 +150,19 @@ subroutine transpmodel(invar, dim_invar, tp_netass, option1)
 
 !       * END OF DAY
 
-        call transpmodel_daily_step(tp_netass, option1)
 
+       if (option1 .eq. 5) then
+           if(nday .eq. 1) allocate( output_mat (21, c_testday ) )
+        call transpmodel_daily_step(tp_netass, option1, output_mat)
+       else
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: I don't see a difference between option1 .eq. 5 and "else". It seems that the following actions are identical. Could you verify and simplify or document?

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 2]
```diff
@@ -1528,6 +1540,90 @@ end subroutine vom_write_dayyear
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+     subroutine vom_save_dayyear (output)
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: Could you add a short comment what this subroutine does?

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 3]
```diff
@@ -1528,6 +1540,90 @@ end subroutine vom_write_dayyear
...
+
+!      if (nday .eq. c_maxday) then
+!       * call subroutine there to get yearly data for the output
+!        call vom_add_yearly()
+!        write(kfile_resultsyearly,'(i6,18e16.6)') nyear, rain_y,       &
+!     &    par_y, srad_y, vd_y / (dayyear(nday)), esoil_y, etm_y,       &
+!     &    etmg_y, assg_y, rlg_y, rrg_y, cpccg_y, tcg_y,                &
+!     &    etmt_y, asst_y, rlt_y, rrt_y, cpcct_y, tct_y
+!      endif
+
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: If the commented out code is not needed any more, it could be removed.

**Review by schymans — CHANGES_REQUESTED** [review]
Great work! Could you summarize briefly what this PR does, for future reference? I also added some comments to the code.

File: VOM_Fortran/VOM-code/transpmodel.f90 (hunk 1 above)
Comment (rcnijzink) [reply VOM_Fortran/VOM-code/transpmodel.f90]: Yes, I changed this already, this was just a temporary solution. The "option1" is now the trigger in the transpmodel_daily_step module.

File: VOM_Fortran/VOM-code/transpmodel.f90 (hunk 2 above)
Comment (rcnijzink) [reply VOM_Fortran/VOM-code/transpmodel.f90]: I added a brief description

File: VOM_Fortran/VOM-code/transpmodel.f90 (hunk 3 above)
Comment (rcnijzink) [reply VOM_Fortran/VOM-code/transpmodel.f90]: Removed commented code

**Comment by schymans** [comment]
Do you know why Travis failed on this? Could you fix it? Thanks!

**Comment by rcnijzink** [comment]
I added a variable "inputpath", which gfortran on travis didn't like in a stop statement, for a reason I really don't know (it was used without problems in the rest of the same subroutine). I adjusted the stop-statements as a workaround. It should run without problems now.

**Comment by rcnijzink** [comment]
I close this pull-request, as it is a pull on the master of my fork. I'll make branches following a more correct git-workflow.

**Comment by schymans** [comment]
OK, thanks!
